lib/ecto/query.ex

```diff
@@ -988,6 +996,11 @@ defmodule Ecto.Query do
     %{query | joins: remaining, aliases: aliases}
   end
 
+  defp do_exclude(%Ecto.Query{} = query, {:windows, window_names}) when is_list(window_names) do
+    remaining = Enum.filter(query.windows, fn {name, _} -> name not in window_names end)
```

Comment:
i have a hard time imagining the number of windows being used is worth creating a set out of `window_names` for this....but please let me know if this is too lazy of an implementation